go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/kanisterio/kanister
 
-go 1.21.9
+go 1.22
+
+toolchain go1.22.0
```

Comment:
why not 1.22.2 ?

Reply:
Seems it was because my dev environment had go1.22.0 when I ran `go get`. Next PR uses 1.22.2 and updates the mod file accordingly